CONTRIBUTING.md

```diff
@@ -42,9 +42,18 @@ Merging Pull Requests (for project contributors with write access)
   - Contributor is using an e-mail address other than the primary GitHub address and wants that preserved in the history. Contributor must be willing to squash
     the commits manually before acceptance.
 
+## Codespaces
```

Comment:
Leave ## Developer Guide as the top level.
Put ### Codespaces and ### VS Code Codespaces under that topic